MANIFEST.in

```diff
@@ -1,2 +1,12 @@
-include README.md
```

Comment:
Is there a reason this is removed?

Reply:
I was wondering why check manifest did not complain about. I assumed that it was implicitly included. 

I have put it back into the MANIFEST.in

Reply:
Thanks!